app/services/waste_carriers_engine/renewal_completion_service.rb

```diff
@@ -97,7 +97,7 @@ def extend_expiry_date
     end
 
     def create_order_item_logs
-      OrderItemLog.create_from_registration(transient_registration)
+      OrderItemLog.create_from_registration(registration)
```

Comment:
Shouldn't the order_item_log be created based on the new order dat in the transient_registration, not what was previously stored on the registration?

Reply:
So the data can be different? I added a spec to check if it would be the same

Reply:
Yes - the registration should have all order items since it was created, so for example the items from the original registration order; some copy cards ordered a few months later; and a renewal order. The transient registration should have only the order items from the current session. I'm fairly sure that copying from the registration (if it has previous order items) would result in duplication of the older ones.